--- configure.ac.link	2010-08-24 07:50:27.000000000 +0000
+++ configure.ac	2010-11-17 07:44:21.000000000 +0000
@@ -265,7 +265,7 @@
 	cygwin*)
 	  PYTHON_LIBS=`$PYTHON -c 'import os,sys;print("-L"+os.path.join(sys.path[[3]],"config")+" -lpython"+sys.version[[:3]]')` ;;
 	*)
-	  PYTHON_LIBS= ;;
+	  PYTHON_LIBS=`python-config --libs` ;;
 	esac
 	AC_SUBST(PYTHON_LIBS)
 	AC_MSG_RESULT([$PYTHON_LIBS])
@@ -384,7 +384,7 @@
       cygwin*)
 	PHP_LIBS="-lphp${PHP_MAJOR_VERSION}lib" ;;
       *)
-	PHP_LIBS= ;;
+	PHP_LIBS="-lphp5_common" ;;
       esac
       AC_SUBST(PHP_LIBS)
       BINDINGS="$BINDINGS php"
@@ -447,7 +447,7 @@
       dnl Cygwin needs to link with -ltcl8.5 or similar.
       TCL_LIBS=
       case $host_os in
-      cygwin*)
+      cygwin*|linux*)
 	AC_CHECK_LIB([tcl$tcl_version], [Tcl_Init],
 		     [TCL_LIBS="-ltcl$tcl_version"],
 		     [AC_MSG_ERROR([libtcl$tcl_version not found])])
@@ -775,7 +775,7 @@
 
 	RUBY_LIBS=
 	case $host_os in
-	cygwin*) RUBY_LIBS=-lruby ;;
+	cygwin*|linux*) RUBY_LIBS=-lruby ;;
 	esac
 	AC_SUBST(RUBY_LIBS)
 
